AllReadyApp/Web-App/AllReady/Areas/Admin/Features/Tasks/EditTaskCommandHandler.cs

```diff
@@ -56,16 +56,25 @@ public EditTaskCommandHandler(AllReadyContext context)
             }
 
             // Add new attachment
-            if (message.Task.NewAttachment != null && !string.IsNullOrEmpty(message.Task.NewAttachment.Name))
+            if (message.Task.NewAttachment != null && !string.IsNullOrEmpty(message.Task.NewAttachment.FileName))
```

Comment:
@DanielSchiavini, yes, please leave the piece that checks first b/c it is optional. I just want to make sure we're validating the incoming attachment(s) more than once. Nothing creates confusion in the codbase quicker than those type of checks.